apps/storybook/stories/APIs/Accelerometer.stories.jsx

```diff
@@ -42,7 +46,9 @@ export class component extends React.Component {
 
   _subscribe = () => {
     this._subscription = Accelerometer.addListener(accelerometerData => {
-      this.setState({ accelerometerData });
+      if (this._isMounted) {
```

Comment:
It seems like this covers up a bug -- the listener should no longer run once `componentWillUnmount` runs. Figure out why the listener is still being run and fix the issue at the source instead of patching over it.